controllers/users.js

```diff
@@ -143,11 +144,31 @@ const postUserPicture = async (req, res) => {
   }
 }
 
+/**
+ * Gets a newly generated chaincode
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getChaincode = (req, res) => {
+  try {
+    const key = uuid.v4()
+    return res.json({
+      message: 'Generated New Chaincode!!',
+      chaincode: key
```

Comment:
How are we associating the code with a specific user? This controller is just generating a UUID v4, that we could have generated on the UI itself if we dont want to associate it with a user... 🤔

Reply:
Good catch @ankurnarkhede. We should store the chaincode in firestore and read the document id as a key (helps prevent potential collisions also).

@pallabez Can you add that as a comment please, so the next person (maybe you?) can remember to make the appropriate changes. Attach the ticket link in the comments please.

Reply:
Got it 👍